keras_nlp/tokenizers/byte_pair_tokenizer.py

```diff
@@ -536,3 +537,57 @@ def _bpe_merge_and_update_cache(self, tokens):
             tokenized_words, axis=1, separator=" "
         )
         self.cache.insert(tokens, tokenized_words)
+
+    @classmethod
+    def from_preset(
+        cls,
+        preset,
+        **kwargs,
+    ):
+        """Instantiate a GPT-2 tokenizer from preset vocabulary and merge rules.
```

Comment:
Please remove GPT-2 language. This should be a generic docstring for BPE. Same for the other two